src/VisualStudio/Core/Def/Venus/ContainedDocument.cs

```diff
@@ -246,10 +250,10 @@ public void UpdateText(SourceText newText)
             }
         }
 
-        private IEnumerable<TextChange> FilterTextChanges(SourceText originalText, List<TextSpan> editorVisibleSpansInOriginal, IReadOnlyList<TextChange> changes)
+        private IEnumerable<TextChange> FilterTextChanges(SourceText originalText, List<TextSpan> editorVisibleSpansInOriginal, IEnumerable<TextChange> changes)
```

Comment:
it's unfortunate we have to widen it to `IEnumerable`, is it possible to keep it a `IReadOnlyList`?

Reply:
Why is that? Seems unnecessary to require a list.